services/wfm/src/main/java/org/openkilda/wfm/topology/cache/CacheBolt.java

```diff
@@ -257,6 +257,10 @@ private void handleSwitchEvent(SwitchInfoData sw, Tuple tuple, String correlatio
                 //emitRerouteCommands(affectedFlows, tuple, correlationId, FlowOperation.UPDATE, reason);
                 break;
 
+            case DELETED:
```

Comment:
I extremely don't like the idea to use `SwitchInfoData` for this operation. `SwitchInfoData` represent (at least now) hardware events produced by FL. Adding this new not hardware state mixes our "control" logic with hardware signals.

You need to create separate set of messages to control switch.

Reply:
Make sense. I'll change it.